packages/web-pkg/src/composables/breadcrumbs/useBreadcrumbsFromPath.ts

```diff
@@ -37,7 +44,10 @@ export const useBreadcrumbsFromPath = () => {
         ...(isAccessible && {
           to: {
             path: '/' + [...current].splice(0, current.length - resource.length + i + 1).join('/'),
-            query: omit(route.query, 'fileId', 'page') // TODO: we need the correct fileId in the query. until we have that we must omit it because otherwise we would correct the path to the one of the (wrong) fileId.
```

Comment:
uhm... don't we still need to omit the `fileId` from the original query? If there is an `ancestor` in the next line you'll add the correct ancestor file id. If there is no ancestor meta data we **must** not provide a `fileId`.

Reply:
Yep, good catch 👍